app/models/comment.rb

```diff
@@ -1,4 +1,10 @@
 class Comment < ApplicationRecord
-  belongs_to :user
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
   belongs_to :post
+
+  after_save :update_comment_counter
+
+  def update_comment_counter
+    post.increment_counter(:comments_counter, 1)
+  end
```

Comment:
- [ ] I think it would be better if your use the ``increment!`` method to update the comments counter as the current one doesn't work on my side. you can update it to something like ``post. increment!(:comments_counter)``